mimesis/providers/numbers.py

```diff
@@ -17,15 +17,18 @@ class Meta:
 
         name = 'numbers'
 
-    def floats(self, n: int = 2) -> List[float]:
-        """Generate a list of random float numbers.
+    def floats(self, start: float = 0, end: float = 1, n: int = 2,
+               rounding: int = 15) -> List[float]:
+        """Generate a list of 10^n random float numbers.
 
+        :param start: Start range.
+        :param end: End range.
         :param n: Raise 10 to the 'n' power.
+        :param rounding: Max number of decimal digits.
         :return: The list of floating-point numbers.
         """
-        nums = [self.random.random()
+        return [self.random.uniform(start, end, rounding)
```

Comment:
Why not just `n` random floats instead of 10^n?

Reply:
Agree, but I was trying to not break the API. Can I change the argument name to `length` or do you think it will break something?

Reply:
Let it be n, but not 10^n. Do not forget about the changelog.